docs/reference/django-support.md

```diff
@@ -24,7 +24,7 @@ For apm-server 6.2+, make sure you use version 2.0 or higher of `elastic-apm`.
 
 
 ::::{note}
-If you use Django with uwsgi, make sure to [enable threads](http://uwsgi-docs.readthedocs.org/en/latest/Options.html#enable-threads).
+If you use Django with uwsgi, make sure to [enable threads](http://uwsgi-docs.readthedocs.org/en/latest/Options.html#enable-threads) (enabled by default since 2.0.27) and [py-call-uwsgi-fork-hooks](https://uwsgi-docs.readthedocs.io/en/latest/Options.html#py-call-uwsgi-fork-hooks).
```

Comment:
Does it make sense to say they need to enable threads and enable py-call-uwsgi-fork-hooks or is there some other verb associated with py-call-uwsgi-fork-hooks? If the former, maybe:

```suggestion
If you use Django with uwsgi, make sure to enable [threads](http://uwsgi-docs.readthedocs.org/en/latest/Options.html#enable-threads) (enabled by default since 2.0.27) and [py-call-uwsgi-fork-hooks](https://uwsgi-docs.readthedocs.io/en/latest/Options.html#py-call-uwsgi-fork-hooks).
```

Reply:
`enable-threads` is the option you need to enable so I think that's the reason the verb is on the link but both versions work for me

Reply:
I'll leave it up to you!